src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/ConfigurationTest.php

```diff
@@ -188,6 +188,27 @@ public function provideInvalidAssetConfigurationTests()
         yield [$createPackageConfig($config), 'You cannot use both "version" and "json_manifest_path" at the same time under "assets" packages.'];
     }
 
+    /**
+     * @expectedException \Symfony\Component\Config\Definition\Exception\InvalidConfigurationException
```

Comment:
this is deprecated in newer phpunit versions. you can use `expectException` like done in `testInvalidAssetsConfiguration`.
I would also recommend to assert the exception message for clarity.